src/vm/i386/asmhelpers.S

```diff
@@ -1162,20 +1162,30 @@ NESTED_END ResolveWorkerChainLookupAsmStub, _TEXT
 NESTED_ENTRY BackPatchWorkerAsmStub, _TEXT, NoHandler
     PROLOG_BEG
     PROLOG_PUSH eax // it may contain siteAddrForRegisterIndirect
+    PROLOG_PUSH ebx
     PROLOG_PUSH ecx
     PROLOG_PUSH edx
     PROLOG_END
 
-    sub     esp, 4     //  for 16 bytes align
+    // Align Stack Frame
+    mov     ebx, esp
+    and     esp, -16
+
+    // Call BackPatchWorkerStaticStub
+    sub     esp, 8     //  16-bytes align padding
```

Comment:
This change looks unnecessarily complicated. We know how much we push, so we know how much we need to sub from esp. And if you do that instead of aligning to 16 bytes and then subtracting 8 right after then we don't need to save esp to ebx and push ebx in the prolog / pop in the epilog.